modules/UI/videolayout/SmallVideo.js

```diff
@@ -479,10 +483,15 @@ SmallVideo.prototype.updateView = function () {
 
     // Determine whether video, avatar or blackness should be displayed
     let displayMode = this.selectDisplayMode();
-    // Show/hide video
-    setVisibility(this.selectVideoElement(), displayMode === DISPLAY_VIDEO);
-    // Show/hide the avatar
-    setVisibility(this.$avatar(), displayMode === DISPLAY_AVATAR);
+    // Show/hide video.
+    UIUtil.setVisibility(   this.selectVideoElement(),
+                            displayMode === DISPLAY_VIDEO);
+    // Show/hide the avatar.
+    UIUtil.setVisibility(   this.$avatar(),
+                            displayMode === DISPLAY_AVATAR);
+    // Show/hide the display name.
+    UIUtil.setVisibility(   this.$displayName(),
+                            displayMode === DISPLAY_BLACKNESS);
```

Comment:
If the display name is now shown instead of "blackness" it would make sense to rename the display mode to DISPLAY_NAME or something... maybe this one is kind of confusing, so DISPLAY_USER_NAME etc.